libraries/helpers.rb

```diff
@@ -269,7 +269,8 @@ def upgrade_enabled?
     end
 
     def systemd?
-      node['init_package'] == 'systemd'
+      ps1 = shell_out('ps --no-headers 1')
+      ps1.stdout.strip.match?(/systemd/)
```

Comment:
Why are we not relying on Ohai to determine the init package here?

Reply:
I believe this was a workaround put in place when ubuntu 14.04 was still being tested -- back when [this bug](https://github.com/chef/ohai/issues/917) was affecting tests.